index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <h2 class="welcome">Welcome</h2>
+            <button class="vector">☰</button>
+        </header>
+        <section class="headline">
+            <h1 class="black">Hey there. I'm David</h1>
+            <h1 class="green">I'm a software developer</h1>
+        </section>
+        <article class="presentation">
+            <h3 class="content"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+        </article>
+        <footer class="sociallinks">
+            <button class="twitter"><i class="fa-brands fa-twitter"></i></button>
+            <button class="linkedin"><i class="fa-brands fa-linkedin-in"></i></button>
+            <button class="medium"><i class="fa-solid fa-m"></i></button>
+            <button class="github"><i class="fa-brands fa-github"></i></button>
+            <button class="angellist"><i class="fa-brands fa-angellist"></i></button>
```

Comment:
- [ ] Please use a `ul` tag for your list of icons and group them, and use `li` per icon; each icon should be inside a link `a` tag . :+1:  It will make it semantically correct and  `a` tag will let you connect your social networks with the icons, using href attribute . :+1: